### ISSUE TITLE
double space, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#709]
following should be okay,

```abap
      EXCEPTIONS
        excecution_error       = 1
        cancelled              = 2
        insert_into_corr_error = 3
        OTHERS                 = 4.
```
